packages/rocketchat-message-snippet/client/tabBar/views/snippetMessage.html

```diff
@@ -0,0 +1,67 @@
+<template name="snippetMessage">
```

Comment:
Of what I can see in atom, it is just tabs. I changed it all on this commit https://github.com/RocketChat/Rocket.Chat/pull/4623/commits/aafa676d19bd949bf3ea1977eeb22430005da4c2